src/components/BookDetails.js

```diff
@@ -18,8 +19,8 @@ const BookDetails = (props) => {
   );
 };
 
-BookDetails.propTypes = {
-  book: PropTypes.node.isRequired,
-};
+// BookDetails.propTypes = {
+//   book: PropTypes.node.isRequired,
+// };
```

Comment:
- [ ] Please remove all commented code from your codebase because it's a bad practice to commit commented-out code. That's what version control is for. If you need that code back, just fetch it from the repo :smile: